applications/contact_information.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
```

Comment:
Maybe we should have one file per app for abstract base models? If we add many of these base / abstract models then file structure gets pretty messy. Also usually in Django projects I have seen filenames describes type of entities the file has (for example models, views, serializers) rather than name of the class it contains.

I suggest something like base_models.py